packages/ndla-ui/src/LinkBlock/LinkBlock.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import React from 'react';
+import styled from '@emotion/styled';
+import SafeLink from '@ndla/safelink';
+import { Forward, CalendarEd } from '@ndla/icons/common';
+import { breakpoints, colors, spacing, mq } from '@ndla/core';
+
+interface Props {
+  title: {
+    title: string;
+    language: string;
+  };
+  date: string;
+  url: string;
+}
+
+const StyledSafeLink = styled(SafeLink)`
+  text-decoration: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: none;
+  color: inherit;
+  border: 1px solid ${colors.brand.lighter};
+  padding: ${spacing.normal};
+  ${mq.range({ from: breakpoints.tabletWide })} {
```

Comment:
Trenger du egentlig dette breakpointet?

Reply:
Trenger breakpointet for å skille ut forward-icon størrelse og focus-visible tenker jeg? Kom gjerne med forslag om du ser annen løsning.